ddtrace/contrib/internal/langchain/patch.py

```diff
@@ -87,135 +83,29 @@ def _extract_model_name(instance: Any) -> Optional[str]:
     return None
 
 
-def _format_api_key(api_key: Union[str, SecretStr]) -> str:
```

Comment:
We should probably remove https://github.com/DataDog/dd-trace-py/pull/13673/files#diff-9dcf3ac6841d165126226f60f9d0878692ae07c014fa03dbfe0e6271f396f08eL72 this as well since it will be unused!